week2/5.2/piece of cake.py

```diff
@@ -0,0 +1,25 @@
+"""
+Peace of cake.
+By: Eliad Karni.
+
+The feature receives a dict of ingredients as keys and its prices as value,
```

Comment:
Like I wrote in another pr, there is not such thing as the feature receives, its not the right use of the word
https://www.computerhope.com/jargon/f/feature.htm